### DISCUSSION TITLE
Remove items

### DISCUSSION THREAD

**Discussion by cb3inco** [bitstower/markets#92] [Q&A] [answered]
Sorry if this is a dumb question, but how does one remove an item in Markets? I don't want the three default options.

**Comment by tomasz-oponowicz** [comment, marked as answer]
hey, simply click `✓`, select an item you want to remove and click the trash icon.

**Reply by cb3inco** [reply]
Thanks, sorry that didn't 'click' with me to chose that when I opened the app up.
